.scala-steward.conf

```diff
@@ -3,3 +3,4 @@ updates.ignore = [
   { groupId = "org.eclipse.platform" }
 ]
 commits.message = "build(deps): Update ${artifactName} from ${currentVersion} to ${nextVersion}"
+updates.limit = 5
```

Comment:
Probably, 5 is still large, but if we run scala-steward frequently, it would create small number of PRs at a time.

Reply:
Yea I was thinking the same thing, the more often we run it, the less we really have to worry about this.